sagemaker-featurestore/feature_store_kms_key_encryption.ipynb

```diff
@@ -0,0 +1,560 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Instantiate a FeatureGroup object for customers_data and orders_data. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sagemaker.feature_store.feature_group import FeatureGroup\n",
+    "\n",
+    "customers_feature_group = FeatureGroup(name=customers_feature_group_name, sagemaker_session=sagemaker_session)\n",
+    "orders_feature_group = FeatureGroup(name=orders_feature_group_name, sagemaker_session=sagemaker_session)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "current_time_sec = int(round(time.time()))\n",
+    "\n",
+    "record_identifier_feature_name = \"customer_id\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Append EventTime feature to your data frame. This parameter is required, and time stamps each data point."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "customer_data[\"EventTime\"] = pd.Series([current_time_sec]*len(customer_data), dtype=\"float64\")\n",
+    "orders_data[\"EventTime\"] = pd.Series([current_time_sec]*len(orders_data), dtype=\"float64\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "customer_data.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "orders_data.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Load feature definitions to your feature group. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "customers_feature_group.load_feature_definitions(data_frame=customer_data)\n",
+    "orders_feature_group.load_feature_definitions(data_frame=orders_data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## How to create an Online or Offline Feature Store that uses your KMS key for encryption?\n",
+    "\n",
+    "Below we create two feature groups, `customers_feature_group` and `orders_feature_group` respectively, and explain how use your KMS key to securely encrypt your data in your online or offline feature store. \n",
+    "\n",
+    "### How to create an Online Feature store with your KMS key? \n",
+    "To encrypt data in your online feature store, set `enable_online_store` to be `True` and specify your KMS key as parameter `online_store_kms_key_id`. You will need to substitute your Account number in `arn:aws:kms:us-east-1:123456789012:key/` replacing `123456789012` with your Account number.\n",
```

Comment:
""To encrypt data in your online feature store, set `enable_online_store` to `True` and specify your KMS key as parameter `online_store_kms_key_id`. You must substitute your account number in `arn:aws:kms:us-east-1:123456789012:key/`, replacing `123456789012` with your account number."